kubernetes/hdfs/values.yaml

```diff
@@ -46,6 +46,7 @@ namenode:
       resources:
         requests:
           storage: 10Gi
+  env: {}
```

Comment:
Perhaps the variable could be renamed to something like `extraEnvs` to be a little more descriptive.

Reply:
Variable name updated.